dss/error.py

```diff
@@ -81,7 +117,16 @@ def wrapper(*args, **kwargs):
             code = "read_only"
             title = "The DSS is currently read-only"
             stacktrace = ""
-            headers = {'Retry-After': 600}
+            headers = {'Retry-After': '600'}
+
+        # These errors may be returned by the chalice app before reaching here.
+        if status in (requests.codes.server_error,         # 500 status code
+                      requests.codes.bad_gateway,          # 502 status code
+                      requests.codes.service_unavailable,  # 503 status code
+                      requests.codes.gateway_timeout       # 504 status code
+                      ) \
+                and not (func.__qualname__.upper() == 'POST' and 'uuid' in kwargs):  # not POST /bundle/{uuid}/checkout
```

Comment:
Can routing information from connexion be used here?

Reply:
I can look into doing that.

Reply:
Done.  Not sure why I didn't do that before, thanks for the catch.